i18n/ja/docusaurus-plugin-content-docs/current/get-involved/join.md

```diff
@@ -1,33 +1,33 @@
-# UNスマートマッパーになろう！
+# 国連スマートマッパーになろう！
 
-## UNスマートマッピングコミュニティに参加する
+## 国連スマートマッピングコミュニティに参加する
 
 ### Discord [![Discord](https://img.shields.io/discord/1180387931614085130?label=&logo=discord&logoColor=ffffff&color=7389D8&labelColor=6A7EC2)](https://discord.gg/B2uCGWGmZ5)
 
-<a href="https://discord.gg/B2uCGWGmZ5" class="button button--tertiary">UNスマートマップDiscordサーバーに参加するには、ここをクリックしてください</a> または以下のQRコードをスキャンしてください:
+<a href="https://discord.gg/B2uCGWGmZ5" class="button button--tertiary">国連スマート地図のDiscordサーバーに参加するには、ここをクリックしてください</a> または以下のQRコードをスキャンしてください。
 
 ![UN Smart Mappers QR Code](/img/unsmartmappersqrcode.png)
 
 Discordサーバーは、質問をしたり、助けを得たり、他のUNスマートマッパーに会うのに最適な場所です。
 
-### GitHubの問題
-私たちが何をしているのかを見て、[UNスマートマップGitHubの問題](https://github.com/unopengis/7/issues)ページで自己紹介をして貢献を始めることができます。
+### GitHub Issues
+[国連スマート地図のGitHubのIssues](https://github.com/unopengis/7/issues)ページで自己紹介をすることで、私たちの活動を見たり、貢献を始めたりすることができます。
 
-## あなたはどのタイプのUNスマートマッパーですか？
+## あなたはどのタイプの国連スマートマッパーですか？
 
-### スチューデント
-GISやマッピングの経験を積みたい学生ですか？ UNスマートマップのチュートリアルをチェックして、今日からマッピングを始めましょう！
+### 学生
```

Comment:
@hfu as per @asahina820's revisions for the Japanese version should we use with the original Kanji version of the personas as defined in https://github.com/UNopenGIS/7/issues/158 ?

Reply:
I like the translation by @asahina820 and so we can go ahead!

Reply:
@albertkun @hfu Thanks for reviewing!
I'm glad my translation was accepted:)